grass7/imagery/i.sentinel/i.sentinel.coverage/i.sentinel.coverage.py

```diff
@@ -159,14 +165,17 @@ def scenename_split(scenename):
             producttype = name_split[2][:3]
             date_string = name_split[4].split('T')[0]
         else:
-            grass.fatal(_("Sensor %s is not supported yet" % name_split[0]))
+            grass.fatal(_(
+                "Sensor {} is not supported yet".format(name_split[0])))
```

Comment:
```suggestion
            grass.fatal(_(
                "Sensor {} is not supported yet").format(name_split[0]))
```

Check also other places where this fix is needed. 

Otherwise I can't really review the functionality.

Reply:
Thanks a lot, it is now adapted accordingly